turbo.json

```diff
@@ -26,21 +22,21 @@
     "dev": {
       "dotEnv": [".env"],
       "env": ["PRAX", "IDB_VERSION", "USDC_ASSET_ID", "MINIFRONT_URL", "PENUMBRA_NODE_PD_URL"],
-      "dependsOn": ["download-keys", "^build"],
+      "dependsOn": ["compile", "^build"],
```

Comment:
here, `compile` should be handled as part of the dependency on build

Reply:
but maybe this is also another candidate for package.json pre/post script

Reply:
handling `compile` as part of the package.json pre-script would fall into an infinite recursive call loop